src/CrowdedMod/Components/ShapeShifterPagingBehaviour.cs

```diff
@@ -19,6 +20,24 @@ public ShapeShifterPagingBehaviour(IntPtr ptr) : base(ptr)
     public IEnumerable<ShapeshifterPanel> Targets => shapeshifterMinigame.potentialVictims.ToArray();
 
     public override int MaxPageIndex => (Targets.Count() - 1) / MaxPerPage;
+    private TextMeshPro PageText = null!;
+
+    public override void Update()
+    {
+        base.Update();
+
+        if (!PageText)
+        {
+            PageText = Instantiate(HudManager.Instance.KillButton.cooldownTimerText, shapeshifterMinigame.transform);
+            PageText.name = "MenuPageCount";
+            PageText.enableWordWrapping = false;
+            PageText.gameObject.SetActive(true);
+            PageText.transform.localPosition = new(4.1f, -2.36f, -51f);
```

Comment:
I really don't like hardcoding offsets

Reply:
changed it so that it's only set when instantiated